executor/src/witgen/mod.rs

```diff
@@ -157,6 +157,14 @@ impl<'a, 'b, T: FieldElement, Q: QueryCallback<T>> WitnessGenerator<'a, 'b, T, Q
         record_end(OUTER_CODE_NAME);
         reset_and_print_profile_summary();
 
+        log::debug!("Publics:");
+        for (name, public_declaration) in self.analyzed.public_declarations_in_source_order() {
+            let poly_name = &public_declaration.referenced_poly_name();
+            let poly_index = public_declaration.index;
+            let value = columns[poly_name][poly_index as usize];
+            log::debug!("  {name:>30}: {value}");
```

Comment:
Maybe we should actually always print output ones at some point? Looks good for now anyway

Reply:
You mean printing with `INFO` log level? I thought about that, but the number of publics in the continuations example is quite large (57, because we have 49 registers and 2 hashes with 4 elements each). So it would be pretty verbose, unfortunately.

I guess you could always reduce the publics to one hash (might be a good idea to reduce verification time anyway), but in that case it wouldn't be that informative anymore...